### PR TITLE
grafana-agent: remove deprecated fields in test

### PR DESCRIPTION
**Pull request by rlankfo**
Signed-off-by: Robbie Lankford <robert.lankford@grafana.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/grafana-agent.rb [hunk 1]
```diff
@@ -61,15 +61,13 @@ def caveats
...
-        http_listen_port: #{port}
-        grpc_listen_port: #{free_port}
     EOS
 
     system "#{bin}/grafana-agentctl", "config-check", "#{testpath}/grafana-agent.yaml"
 
     fork do
       exec bin/"grafana-agent", "-config.file=#{testpath}/grafana-agent.yaml",
-        "-metrics.wal-directory=#{testpath}/wal"
+        "-metrics.wal-directory=#{testpath}/wal", "-server.http.address=127.0.0.1:#{port}"
```
Comment (rfratto) [inline Formula/grafana-agent.rb:70]: Shouldn't this also set grpc? `--server.grpc.address=127.0.0.1:#{free_port}`

Comment (rlankfo) [reply Formula/grafana-agent.rb:70]: it isn't needed for the test -- the test only curls the http port /metrics endpoint

**Comment by SMillerDev** [comment]
Could you rebase this?

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @rlankfo!

**Pull request merged**